image.yaml

```diff
@@ -99,7 +99,11 @@ modules:
       repositories:
           - name: cct_module
             git:
-                  url: https://github.com/jboss-openshift/cct_module.git
+                  url: https://github.com/jboss-openshift/cct_module
```

Comment:
Should the .git have been dropped?

Reply:
Thanks. Probably not, but concreate still seems to work. I'll fix.